src/components/AvatarList.astro

```diff
@@ -2,6 +2,7 @@
 // fetch all commits for just this page's path
 const { path } = Astro.props;
 const url = `https://api.github.com/repos/snowpackjs/astro-docs/commits?path=${path}`;
+const blameURL = `https://github.com/snowpackjs/astro-docs/blame/main/${path}`;
```

Comment:
what about the commit log instead: `https://github.com/snowpackjs/astro-docs/commits/main/...`

I don't have a too strong opinion, but sometimes the blame is noisy and might not highlight the authors

Reply:
I like the blame view, though I agree it is noisy. I changed to using `commitsURL`.